conneg-by-ap/index.html

```diff
@@ -230,8 +231,47 @@ <h2>Requirements</h2>
       <div class="req-response">
         <p class="respTitle">RESPONSE FOR Req. 73</p>
         <p>
-
+          This requirement is addressed by suggesting how an Internet resource in general, rather than specifically a
+          <code>dcat:Dataset</code> or <code>dcat:Distribution</code> should list profiles it implements. A DCAT-
+          specific solution should not be any different from the general case.
+        </p>
+        <div class="note" title="">Profile definition and constraints on properties are not addressed here. See
+          [[PROF-GUIDE]].</div>
+        <p>
+          This requirement is taken to mean "create a way to list the profiles implemented by a Internet resource for
+          <em>humans</em> and <em>machines</em> to use". For the former (humans), the options are:
         </p>
+        <ul>
+          <li>
+            discovery via HTML representation
+            <ul>
+              <li>see [[PROF-GUIDE]]</li>
+              <li>the approach recommended is to provide an <em>alternate profiles</em> resource for the original
+                resource located at <code>RESOURCE_URI + ?_profile=alternates</code> which lists, at a minimum, the
```

Comment:
I think it's too early to say that query strings with `profile=alternates` is the _recommended_ approach. IMHO it's one possible approach.

Reply:
Demo answer only! To show what answers could look like, not what this one actually is.